### DISCUSSION TITLE
Barcode Scanner – Search for products using barcode scan -For single-vendor or multi-vendor application

### DISCUSSION THREAD

**Discussion by tadamk** [inspireui/support#5323] [Working on Features]
Barcode Scanner – Search for products using barcode scan (required plugin YITH Woocommerce Barcodes and QR Codes).For single-vendor or multi-vendor application

https://yithemes.com/themes/plugins/yith-woocommerce-barcodes-and-qr-codes/

The application would be very cool if this feature was added

<img width="223" alt="bcode" src="https://user-images.githubusercontent.com/89693592/133945313-f1225dd9-de65-4af5-bef0-bbb9e95939c0.png">
<img width="524" alt="Untitled1111" src="https://user-images.githubusercontent.com/89693592/133945521-3effd52f-07f0-4c90-a0be-17ca0a450b85.png">

**Comment by revoru** [comment]
This is a very important feature for grocery and convenience store owners

**Reply by Haniel5** [reply]
Hi, this feature is available on version 2.5.3 on FluxStore WooCommerce, FluxStore Multi Vendors, and FluxStore Pro (woocommerce) as our video https://youtu.be/yC-Z3hZ6hu4 - Very Best!

**Comment by tadamk** [comment]
and working dokan and wcfm

**Reply by Haniel5** [reply]
Hi, this feature is available on version 2.5.3 on FluxStore WooCommerce, FluxStore Multi Vendors, and FluxStore Pro (woocommerce) as our video https://youtu.be/yC-Z3hZ6hu4 - Very Best!

**Comment by aliadnandeveloper** [comment]
/

**Reply by Thedeep1** [reply]
Wow

**Reply by Haniel5** [reply]
Hi, this feature is available on version 2.5.3 on FluxStore WooCommerce, FluxStore Multi Vendors, and FluxStore Pro (woocommerce) as our video https://youtu.be/yC-Z3hZ6hu4 - Very Best!

**Comment by tarik8844** [comment]
i am searching to implement it

**Reply by Haniel5** [reply]
Hi, this feature is available on version 2.5.3 on FluxStore WooCommerce, FluxStore Multi Vendors, and FluxStore Pro (woocommerce) as our video https://youtu.be/yC-Z3hZ6hu4 - Very Best!

**Comment by aliadnandeveloper** [comment]
and working dokan and wcfm

**Reply by Haniel5** [reply]
Hi, this feature is available on version 2.5.3 on FluxStore WooCommerce, FluxStore Multi Vendors, and FluxStore Pro (woocommerce) as our video https://youtu.be/yC-Z3hZ6hu4 - Very Best!

**Comment by Haniel5** [comment]
Hi, this feature is available on version 2.5.3 on FluxStore WooCommerce, FluxStore Multi Vendors, and FluxStore Pro (woocommerce) as our video https://youtu.be/yC-Z3hZ6hu4 - Very Best!

**Reply by qadyy** [reply]
Hello Haniel5, I hope all is well.
When can this feature support SKU or custom field? since in our retail system, we already have hundreds of generated barcodes and we'd like to use them without creating SKU from product ID. Any workarounds?
Thanks

**Reply by ChiKaLiO** [reply]
any solution ? using the id is bit weird

**Reply by chungxon** [reply]
Hi @qadyy and @ChiKaLiO,

Scan product feature already supported ID, SKU, and custom field. Please upgrade MStore API Plugin to the latest version and check again.

Regards,
